<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<html lang="en">
  <head>
    <meta charset="utf-8">
<!--
    <link rel="stylesheet" type="text/css" href="http://epigenomegateway.wustl.edu/browser/style.css" />
    <script type="text/javascript" src="http://epigenomegateway.wustl.edu/browser/js/base.js"></script>
    <script type="text/javascript" src="http://epigenomegateway.wustl.edu/browser/js/personality.js"></script>
    <script type="text/javascript" src="http://epigenomegateway.wustl.edu/browser/js/embed.js"></script>
-->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    
    <link href="./report/jquery.treetable.css" rel="stylesheet" type="text/css" />
    <link href="./report/jquery.treetable.theme.default.css" rel="stylesheet" />
    <script src="./report/jquery.treetable.js"></script>

    <script type="text/javascript">
      $( function(){
      	$("#filetable").treetable({ 
      		expandable: true,
      		onNodeCollapse: function() {
      		},
      		onNodeExpand: function() {
      		}
      	});
      });
    </script>
  </head>
  <body>
<div id='title'><b>Title: output_treatedMB6Pen</b><br><p>Report generated at Wed Aug 22 17:39:56 EDT 2018 <br></p></div><div id='cmd_line_args'><b>Command line arguments</b><br><p>-true_rep -enable_idr -use_sambamba_markdup true -mem_bwt2 30G -mem_dedup 20G -mem_shuf 20G -mem_macs2 40G -mem_trim 20G -memory 40G --unlimited_mem_wt TRUE -species hg19 -ref_fa /mnt/work1/users/carlsgroup/workinprogres/nibeh/ATAC-seq-pipeline/atac_dnase_pipelines/genome_data/hg19/male.hg19.fa -bwt2_idx /mnt/work1/users/carlsgroup/workinprogres/nibeh/ATAC-seq-pipeline/atac_dnase_pipelines/INDEX -nth 6 -chrsz /mnt/work1/users/carlsgroup/workinprogres/nibeh/ATAC-seq-pipeline/atac_dnase_pipelines/genome_data/hg19/sizes.genome -gensz hs -se -fastq1 /mnt/work1/users/carlsgroup/workinprogres/nibeh/180719_D00165_0213_ACCL7AANXX_180801_D00165_0215_ACCJMHANXX_Rashim/180719_D00165_0213_ACCL7AANXX/7_S23_L006_R1_001.fastq.gz -fastq2 /mnt/work1/users/carlsgroup/workinprogres/nibeh/180719_D00165_0213_ACCL7AANXX_180801_D00165_0215_ACCJMHANXX_Rashim/180719_D00165_0213_ACCL7AANXX/8_S24_L006_R1_001.fastq.gz -out_dir /mnt/work1/users/carlsgroup/workinprogres/nibeh/atac_dnase_pipelines5/output_treatedMB6Pen</p></div><div id='latest_git_commit'><b>Pipeline version</b><br><p>Latest git commit SHA1: <a target=_blank, href=https://github.com/kundajelab/atac_dnase_pipelines/commit/a0ff6c2fd4537f6e5af5bf2682ad7436fa4b6314>a0ff6c2fd4537f6e5af5bf2682ad7436fa4b6314</a> (Thu Sep 21 19:05:31 2017)
</p></div><br>
<div id='dirs_and_files'><b>Directories and files</b><table id='filetable'> <caption><a href='#' onclick="jQuery('#filetable').treetable('expandAll'); return false;">Expand all</a> &nbsp&nbsp<a href='#' onclick="jQuery('#filetable').treetable('collapseAll'); return false;">Collapse all</a></caption> <thead><tr><th>Files</th><th>Path</th></tr></thead> <tbody><tr data-tt-id='Raw reads'><td> Raw reads </td><td><br></td></tr><tr data-tt-id='Raw reads/Replicate 1' data-tt-parent-id='Raw reads'><td> Replicate 1 </td><td><br></td></tr><tr data-tt-id='Raw reads/Replicate 1/Fastq (59M)' data-tt-parent-id='Raw reads/Replicate 1'><td> Fastq (59M) </td><td>/mnt/work1/users/carlsgroup/workinprogres/nibeh/180719_D00165_0213_ACCL7AANXX_180801_D00165_0215_ACCJMHANXX_Rashim/180719_D00165_0213_ACCL7AANXX/7_S23_L006_R1_001.fastq.gz<br></td></tr><tr data-tt-id='Raw reads/Replicate 2' data-tt-parent-id='Raw reads'><td> Replicate 2 </td><td><br></td></tr><tr data-tt-id='Raw reads/Replicate 2/Fastq (126M)' data-tt-parent-id='Raw reads/Replicate 2'><td> Fastq (126M) </td><td>/mnt/work1/users/carlsgroup/workinprogres/nibeh/180719_D00165_0213_ACCL7AANXX_180801_D00165_0215_ACCJMHANXX_Rashim/180719_D00165_0213_ACCL7AANXX/8_S24_L006_R1_001.fastq.gz<br></td></tr><tr data-tt-id='Alignment'><td> Alignment </td><td><br></td></tr><tr data-tt-id='Alignment/Replicate 1' data-tt-parent-id='Alignment'><td> Replicate 1 </td><td><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Bam (59M)' data-tt-parent-id='Alignment/Replicate 1'><td> Bam (59M) </td><td><a href='./align/rep1/7_S23_L006_R1_001.bam' target='_blank'>./align/rep1/7_S23_L006_R1_001.bam</a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Filtered & deduped bam (15M)' data-tt-parent-id='Alignment/Replicate 1'><td> Filtered & deduped bam (15M) </td><td><a href='./align/rep1/7_S23_L006_R1_001.nodup.bam' target='_blank'>./align/rep1/7_S23_L006_R1_001.nodup.bam</a><br></td></tr><tr data-tt-id='Alignment/Replicate 1/Tag-align' data-tt-parent-id='Alignment/Replicate 1'><td> Tag-align </td><td><a href='./align/rep1/7_S23_L006_R1_001.nodup.tn5.tagAlign.gz' target='_blank'>./align/rep1/7_S23_L006_R1_001.nodup.tn5.tagAlign.gz</a><br></td></tr><tr data-tt-id='Alignment/Replicate 2' data-tt-parent-id='Alignment'><td> Replicate 2 </td><td><br></td></tr><tr data-tt-id='Alignment/Replicate 2/Bam (126M)' data-tt-parent-id='Alignment/Replicate 2'><td> Bam (126M) </td><td><a href='./align/rep2/8_S24_L006_R1_001.bam' target='_blank'>./align/rep2/8_S24_L006_R1_001.bam</a><br></td></tr><tr data-tt-id='Alignment/Replicate 2/Filtered & deduped bam (32M)' data-tt-parent-id='Alignment/Replicate 2'><td> Filtered & deduped bam (32M) </td><td><a href='./align/rep2/8_S24_L006_R1_001.nodup.bam' target='_blank'>./align/rep2/8_S24_L006_R1_001.nodup.bam</a><br></td></tr><tr data-tt-id='Alignment/Replicate 2/Tag-align' data-tt-parent-id='Alignment/Replicate 2'><td> Tag-align </td><td><a href='./align/rep2/8_S24_L006_R1_001.nodup.tn5.tagAlign.gz' target='_blank'>./align/rep2/8_S24_L006_R1_001.nodup.tn5.tagAlign.gz</a><br></td></tr><tr data-tt-id='Alignment/Pooled replicate' data-tt-parent-id='Alignment'><td> Pooled replicate </td><td><br></td></tr><tr data-tt-id='Alignment/Pooled replicate/Tag-align' data-tt-parent-id='Alignment/Pooled replicate'><td> Tag-align </td><td><a href='./align/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.tagAlign.gz' target='_blank'>./align/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.tagAlign.gz</a><br></td></tr><tr data-tt-id='Signal tracks'><td> Signal tracks </td><td><br></td></tr><tr data-tt-id='Signal tracks/MACS2' data-tt-parent-id='Signal tracks'><td> MACS2 </td><td><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 1' data-tt-parent-id='Signal tracks/MACS2'><td> Replicate 1 </td><td><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 1/P-value' data-tt-parent-id='Signal tracks/MACS2/Replicate 1'><td> P-value </td><td><a href='./signal/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.pval.signal.bigwig' target='_blank'>./signal/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.pval.signal.bigwig</a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 1/Fold enrichment' data-tt-parent-id='Signal tracks/MACS2/Replicate 1'><td> Fold enrichment </td><td><a href='./signal/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.fc.signal.bigwig' target='_blank'>./signal/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.fc.signal.bigwig</a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 2' data-tt-parent-id='Signal tracks/MACS2'><td> Replicate 2 </td><td><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 2/P-value' data-tt-parent-id='Signal tracks/MACS2/Replicate 2'><td> P-value </td><td><a href='./signal/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.pval.signal.bigwig' target='_blank'>./signal/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.pval.signal.bigwig</a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Replicate 2/Fold enrichment' data-tt-parent-id='Signal tracks/MACS2/Replicate 2'><td> Fold enrichment </td><td><a href='./signal/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.fc.signal.bigwig' target='_blank'>./signal/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.fc.signal.bigwig</a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Pooled replicate' data-tt-parent-id='Signal tracks/MACS2'><td> Pooled replicate </td><td><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Pooled replicate/P-value' data-tt-parent-id='Signal tracks/MACS2/Pooled replicate'><td> P-value </td><td><a href='./signal/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.pval.signal.bigwig' target='_blank'>./signal/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.pval.signal.bigwig</a><br></td></tr><tr data-tt-id='Signal tracks/MACS2/Pooled replicate/Fold enrichment' data-tt-parent-id='Signal tracks/MACS2/Pooled replicate'><td> Fold enrichment </td><td><a href='./signal/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.fc.signal.bigwig' target='_blank'>./signal/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.fc.signal.bigwig</a><br></td></tr><tr data-tt-id='Peaks'><td> Peaks </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2' data-tt-parent-id='Peaks'><td> MACS2 </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1' data-tt-parent-id='Peaks/MACS2'><td> Replicate 1 </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Narrow peak (p-val thresh=0.01)' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Narrow peak (p-val thresh=0.01) </td><td><a href='./peak/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.narrowPeak.gz' target='_blank'>./peak/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 1/Filtered narrow peak' data-tt-parent-id='Peaks/MACS2/Replicate 1'><td> Filtered narrow peak </td><td><a href='./peak/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.500K.pval0.01.narrowPeak.gz' target='_blank'>./peak/macs2/rep1/7_S23_L006_R1_001.nodup.tn5.pf.500K.pval0.01.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 2' data-tt-parent-id='Peaks/MACS2'><td> Replicate 2 </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 2/Narrow peak (p-val thresh=0.01)' data-tt-parent-id='Peaks/MACS2/Replicate 2'><td> Narrow peak (p-val thresh=0.01) </td><td><a href='./peak/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.narrowPeak.gz' target='_blank'>./peak/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Replicate 2/Filtered narrow peak' data-tt-parent-id='Peaks/MACS2/Replicate 2'><td> Filtered narrow peak </td><td><a href='./peak/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.500K.pval0.01.narrowPeak.gz' target='_blank'>./peak/macs2/rep2/8_S24_L006_R1_001.nodup.tn5.pf.500K.pval0.01.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pooled replicate' data-tt-parent-id='Peaks/MACS2'><td> Pooled replicate </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/Pooled replicate/Narrow peak (p-val thresh=.01)' data-tt-parent-id='Peaks/MACS2/Pooled replicate'><td> Narrow peak (p-val thresh=.01) </td><td><a href='./peak/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.narrowPeak.gz' target='_blank'>./peak/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Pooled replicate/Filtered narrow peak' data-tt-parent-id='Peaks/MACS2/Pooled replicate'><td> Filtered narrow peak </td><td><a href='./peak/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.500K.pval0.01.narrowPeak.gz' target='_blank'>./peak/macs2/pooled_rep/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.500K.pval0.01.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/Naive overlap' data-tt-parent-id='Peaks/MACS2'><td> Naive overlap </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/Naive overlap/Narrow peak' data-tt-parent-id='Peaks/MACS2/Naive overlap'><td> Narrow peak </td><td><a href='./peak/macs2/overlap/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.500K.pval0.01.naive_overlap.narrowPeak.gz' target='_blank'>./peak/macs2/overlap/7_S23_L006_R1_001.nodup.tn5_8_S24_L006_R1_001.nodup.tn5.pf.500K.pval0.01.naive_overlap.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR' data-tt-parent-id='Peaks/MACS2'><td> IDR </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/True replicates' data-tt-parent-id='Peaks/MACS2/IDR'><td> True replicates </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/True replicates/Rep. 1 vs. Rep. 2' data-tt-parent-id='Peaks/MACS2/IDR/True replicates'><td> Rep. 1 vs. Rep. 2 </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/True replicates/Rep. 1 vs. Rep. 2/IDR peak' data-tt-parent-id='Peaks/MACS2/IDR/True replicates/Rep. 1 vs. Rep. 2'><td> IDR peak </td><td><a href='./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.gz' target='_blank'>./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/Optimal set' data-tt-parent-id='Peaks/MACS2/IDR'><td> Optimal set </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/Optimal set/IDR peak' data-tt-parent-id='Peaks/MACS2/IDR/Optimal set'><td> IDR peak </td><td><a href='./peak/macs2/idr/optimal_set/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.gz' target='_blank'>./peak/macs2/idr/optimal_set/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/Conservative set' data-tt-parent-id='Peaks/MACS2/IDR'><td> Conservative set </td><td><br></td></tr><tr data-tt-id='Peaks/MACS2/IDR/Conservative set/IDR peak' data-tt-parent-id='Peaks/MACS2/IDR/Conservative set'><td> IDR peak </td><td><a href='./peak/macs2/idr/conservative_set/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.gz' target='_blank'>./peak/macs2/idr/conservative_set/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.gz</a><br></td></tr><tr data-tt-id='QC and logs'><td> QC and logs </td><td><br></td></tr><tr data-tt-id='QC and logs/Replicate 1' data-tt-parent-id='QC and logs'><td> Replicate 1 </td><td><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Filtered flagstat log' data-tt-parent-id='QC and logs/Replicate 1'><td> Filtered flagstat log </td><td><a href='./qc/rep1/7_S23_L006_R1_001.nodup.flagstat.qc' target='_blank'>./qc/rep1/7_S23_L006_R1_001.nodup.flagstat.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Bowtie2 map. log' data-tt-parent-id='QC and logs/Replicate 1'><td> Bowtie2 map. log </td><td><a href='./qc/rep1/7_S23_L006_R1_001.align.log' target='_blank'>./qc/rep1/7_S23_L006_R1_001.align.log</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Dedup. log' data-tt-parent-id='QC and logs/Replicate 1'><td> Dedup. log </td><td><a href='./qc/rep1/7_S23_L006_R1_001.dup.qc' target='_blank'>./qc/rep1/7_S23_L006_R1_001.dup.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Bowtie2 map. flagstat log' data-tt-parent-id='QC and logs/Replicate 1'><td> Bowtie2 map. flagstat log </td><td><a href='./qc/rep1/7_S23_L006_R1_001.flagstat.qc' target='_blank'>./qc/rep1/7_S23_L006_R1_001.flagstat.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/PBC log' data-tt-parent-id='QC and logs/Replicate 1'><td> PBC log </td><td><a href='./qc/rep1/7_S23_L006_R1_001.nodup.pbc.qc' target='_blank'>./qc/rep1/7_S23_L006_R1_001.nodup.pbc.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 1/Cross-corr. plot' data-tt-parent-id='QC and logs/Replicate 1'><td> Cross-corr. plot </td><td><a href='./qc/rep1/7_S23_L006_R1_001.nodup.25M.cc.plot.pdf' target='_blank'>./qc/rep1/7_S23_L006_R1_001.nodup.25M.cc.plot.pdf</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 2' data-tt-parent-id='QC and logs'><td> Replicate 2 </td><td><br></td></tr><tr data-tt-id='QC and logs/Replicate 2/Filtered flagstat log' data-tt-parent-id='QC and logs/Replicate 2'><td> Filtered flagstat log </td><td><a href='./qc/rep2/8_S24_L006_R1_001.nodup.flagstat.qc' target='_blank'>./qc/rep2/8_S24_L006_R1_001.nodup.flagstat.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 2/Bowtie2 map. log' data-tt-parent-id='QC and logs/Replicate 2'><td> Bowtie2 map. log </td><td><a href='./qc/rep2/8_S24_L006_R1_001.align.log' target='_blank'>./qc/rep2/8_S24_L006_R1_001.align.log</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 2/Dedup. log' data-tt-parent-id='QC and logs/Replicate 2'><td> Dedup. log </td><td><a href='./qc/rep2/8_S24_L006_R1_001.dup.qc' target='_blank'>./qc/rep2/8_S24_L006_R1_001.dup.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 2/Bowtie2 map. flagstat log' data-tt-parent-id='QC and logs/Replicate 2'><td> Bowtie2 map. flagstat log </td><td><a href='./qc/rep2/8_S24_L006_R1_001.flagstat.qc' target='_blank'>./qc/rep2/8_S24_L006_R1_001.flagstat.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 2/PBC log' data-tt-parent-id='QC and logs/Replicate 2'><td> PBC log </td><td><a href='./qc/rep2/8_S24_L006_R1_001.nodup.pbc.qc' target='_blank'>./qc/rep2/8_S24_L006_R1_001.nodup.pbc.qc</a><br></td></tr><tr data-tt-id='QC and logs/Replicate 2/Cross-corr. plot' data-tt-parent-id='QC and logs/Replicate 2'><td> Cross-corr. plot </td><td><a href='./qc/rep2/8_S24_L006_R1_001.nodup.25M.cc.plot.pdf' target='_blank'>./qc/rep2/8_S24_L006_R1_001.nodup.25M.cc.plot.pdf</a><br></td></tr><tr data-tt-id='QC and logs/IDR' data-tt-parent-id='QC and logs'><td> IDR </td><td><br></td></tr><tr data-tt-id='QC and logs/IDR/IDR QC log' data-tt-parent-id='QC and logs/IDR'><td> IDR QC log </td><td><a href='./qc/output_treatedMB6Pen_IDR_final.qc' target='_blank'>./qc/output_treatedMB6Pen_IDR_final.qc</a><br></td></tr><tr data-tt-id='QC and logs/IDR/True replicates' data-tt-parent-id='QC and logs/IDR'><td> True replicates </td><td><br></td></tr><tr data-tt-id='QC and logs/IDR/True replicates/Rep. 1 vs. Rep. 2' data-tt-parent-id='QC and logs/IDR/True replicates'><td> Rep. 1 vs. Rep. 2 </td><td><br></td></tr><tr data-tt-id='QC and logs/IDR/True replicates/Rep. 1 vs. Rep. 2/FRiP' data-tt-parent-id='QC and logs/IDR/True replicates/Rep. 1 vs. Rep. 2'><td> FRiP </td><td><a href='./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.FRiP.qc' target='_blank'>./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.IDR0.1.narrowPeak.FRiP.qc</a><br></td></tr><tr data-tt-id='QC and logs/IDR/True replicates/Rep. 1 vs. Rep. 2/IDR plot' data-tt-parent-id='QC and logs/IDR/True replicates/Rep. 1 vs. Rep. 2'><td> IDR plot </td><td><a href='./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.unthresholded-peaks.txt.png' target='_blank'>./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.unthresholded-peaks.txt.png</a><br></td></tr></tbody></table></div><br>
<div id='viz'><b>Visualization (add <i>-url_base [URL_ROOT_DIR_FOR_OUT_DIR]</i> to the command line.) </b><div style='padding-left: 20px'><a href=http://epigenomegateway.wustl.edu/browser/?genome=hg19&tknamewidth=275&datahub=/./output_treatedMB6Pen_tracks.json target='_blank'>Visualize</a>&nbsp&nbsp<a href=./output_treatedMB6Pen_tracks.json target='_blank'>JSON (datahub)</a></div></div><br>

<button id='btn_toggle'>Toggle workflow</button><br><br>
<script>$('#btn_toggle').click(function() {$('#graph').toggle();});</script><div id='graph'><b>Workflow diagram</b>&nbsp<small>(g./b. peak : gapped/broad peak from MACS2, n. peak : narrow peak from MACS2, r. peak : relaxed peak from SPP)</small><br><object data='./report/output_treatedMB6Pen_workflow.svg'></object></div><br></div><br>
<div id='atac_qc'><div style='float:left'><div id='table_Flagstat-(raw)-QC'><b>Flagstat (raw) QC</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>&nbsp</th><th nowrap align=left><a href='./qc/rep1/7_S23_L006_R1_001.flagstat.qc' target='_blank'>rep1</a><br></th><th nowrap align=left><a href='./qc/rep2/8_S24_L006_R1_001.flagstat.qc' target='_blank'>rep2</a><br></th></tr>
<tr><th nowrap align=left>Total</th><td>59523771</td><td>126937869</td></tr>
<tr><th nowrap align=left>Total(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Dupes</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Dupes(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Mapped</th><td>59032296</td><td>126177131</td></tr>
<tr><th nowrap align=left>Mapped(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>% Mapped</th><td>99.17</td><td>99.40</td></tr>
<tr><th nowrap align=left>Paired</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Paired(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read1</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read1(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read2</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read2(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Properly Paired</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Properly Paired(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>% Properly Paired</th><td>N/A</td><td>N/A</td></tr>
<tr><th nowrap align=left>With itself</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>With itself(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Singletons</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Singletons(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>% Singleton</th><td>N/A</td><td>N/A</td></tr>
<tr><th nowrap align=left>Diff. Chroms</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Diff. Chroms (QC-failed)</th><td>0</td><td>0</td></tr>
</table></div></div><div style='float:left'><div id='table_Dup.-QC'><b>Dup. QC</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>&nbsp</th><th nowrap align=left><a href='./qc/rep1/7_S23_L006_R1_001.dup.qc' target='_blank'>rep1</a><br></th><th nowrap align=left><a href='./qc/rep2/8_S24_L006_R1_001.dup.qc' target='_blank'>rep2</a><br></th></tr>
<tr><th nowrap align=left>Unpaired Reads</th><td>41210940</td><td>91774022</td></tr>
<tr><th nowrap align=left>Paired Reads</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Unmapped Reads</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Unpaired Dupes</th><td>25565668</td><td>59023400</td></tr>
<tr><th nowrap align=left>Paired Dupes</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Paired Opt. Dupes</th><td>N/A</td><td>N/A</td></tr>
<tr><th nowrap align=left>% Dupes/100</th><td>0.620361</td><td>0.643138</td></tr>
</table></div></div><div style='float:left'><div id='table_Flagstat-(filtered)-QC'><b>Flagstat (filtered) QC</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>&nbsp</th><th nowrap align=left><a href='./qc/rep1/7_S23_L006_R1_001.nodup.flagstat.qc' target='_blank'>rep1</a><br></th><th nowrap align=left><a href='./qc/rep2/8_S24_L006_R1_001.nodup.flagstat.qc' target='_blank'>rep2</a><br></th></tr>
<tr><th nowrap align=left>Total</th><td>15645272</td><td>32750622</td></tr>
<tr><th nowrap align=left>Total(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Dupes</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Dupes(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Mapped</th><td>15645272</td><td>32750622</td></tr>
<tr><th nowrap align=left>Mapped(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>% Mapped</th><td>100.00</td><td>100.00</td></tr>
<tr><th nowrap align=left>Paired</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Paired(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read1</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read1(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read2</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Read2(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Properly Paired</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Properly Paired(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>% Properly Paired</th><td>N/A</td><td>N/A</td></tr>
<tr><th nowrap align=left>With itself</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>With itself(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Singletons</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Singletons(QC-failed)</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>% Singleton</th><td>N/A</td><td>N/A</td></tr>
<tr><th nowrap align=left>Diff. Chroms</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Diff. Chroms (QC-failed)</th><td>0</td><td>0</td></tr>
</table></div></div><div style='float:left'><div id='table_Library-Complexity-QC'><b>Library Complexity QC</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>&nbsp</th><th nowrap align=left><a href='./qc/rep1/7_S23_L006_R1_001.nodup.pbc.qc' target='_blank'>rep1</a><br></th><th nowrap align=left><a href='./qc/rep2/8_S24_L006_R1_001.nodup.pbc.qc' target='_blank'>rep2</a><br></th></tr>
<tr><th nowrap align=left>Total Reads</th><td>18103608</td><td>45297344</td></tr>
<tr><th nowrap align=left>Distinct Reads</th><td>15736847</td><td>33011972</td></tr>
<tr><th nowrap align=left>One Read</th><td>14210555</td><td>25208055</td></tr>
<tr><th nowrap align=left>Two Reads</th><td>1232312</td><td>5611379</td></tr>
<tr><th nowrap align=left>NRF = Distinct/Total</th><td>0.869266</td><td>0.728784</td></tr>
<tr><th nowrap align=left>PBC1 = OneRead/Distinct</th><td>0.903012</td><td>0.763603</td></tr>
<tr><th nowrap align=left>PBC2 = OneRead/TwoReads</th><td>11.531621</td><td>4.492310</td></tr>
</table></div><div id='help-pbc'><p>NRF (non redundant fraction) <br>PBC1 (PCR Bottleneck coefficient 1) <br>PBC2 (PCR Bottleneck coefficient 2) <br>PBC1 is the primary measure. Provisionally <br><ul><li>0-0.5 is severe bottlenecking</li><li>0.5-0.8 is moderate bottlenecking </li><li>0.8-0.9 is mild bottlenecking </li><li>0.9-1.0 is no bottlenecking </li></ul></p></div></div><div style='float:left'><div id='table_Enrichment-QC-(strand-cross-correlation-measures)'><b>Enrichment QC (strand cross-correlation measures)</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>&nbsp</th><th nowrap align=left><a href='./qc/rep1/7_S23_L006_R1_001.nodup.25M.cc.qc' target='_blank'>rep1</a><br></th><th nowrap align=left><a href='./qc/rep2/8_S24_L006_R1_001.nodup.25M.cc.qc' target='_blank'>rep2</a><br></th></tr>
<tr><th nowrap align=left>Reads</th><td>15621744</td><td>25000000</td></tr>
<tr><th nowrap align=left>Est. Fragment Len.</th><td>0</td><td>0</td></tr>
<tr><th nowrap align=left>Corr. Est. Fragment Len.</th><td>0.233053442338985</td><td>0.296937182923509</td></tr>
<tr><th nowrap align=left>Phantom Peak</th><td>55</td><td>55</td></tr>
<tr><th nowrap align=left>Corr. Phantom Peak</th><td>0.2376072</td><td>0.2976381</td></tr>
<tr><th nowrap align=left>Argmin. Corr.</th><td>1500</td><td>1500</td></tr>
<tr><th nowrap align=left>Min. Corr.</th><td>0.1754485</td><td>0.2542398</td></tr>
<tr><th nowrap align=left>NSC</th><td>1.328329</td><td>1.167942</td></tr>
<tr><th nowrap align=left>RSC</th><td>0.9267392</td><td>0.9838498</td></tr>
</table></div><div id='help-xcor'><p>NOTE: Reads from replicates are subsampled to a max of 25M <br><ul><li>Normalized strand cross-correlation coefficient (NSC) = col9 in outFile </li><li>Relative strand cross-correlation coefficient (RSC) = col10 in outFile </li><li>Estimated fragment length = col3 in outFile, take the top value </li></ul></p><br></div><figure style='display: inline-block;'><img src='./qc/rep1/7_S23_L006_R1_001.nodup.25M.cc.plot.png' width='500'><figcaption style='text-align: center;'><b>rep1</b></figcaption></figure>&nbsp<figure style='display: inline-block;'><img src='./qc/rep2/8_S24_L006_R1_001.nodup.25M.cc.plot.png' width='500'><figcaption style='text-align: center;'><b>rep2</b></figcaption></figure>&nbsp<div style='float:left'><div id='table_Number-of-peaks'><b>Number of peaks</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>rep1</th><td>184951</td></tr>
<tr><th nowrap align=left>rep2</th><td>234041</td></tr>
<tr><th nowrap align=left>pooled</th><td>329012</td></tr>
<tr><th nowrap align=left>overlap</th><td>155613</td></tr>
</table></div><div id='help-num-peaks'><p><ul><li>ppr1: Raw peaks called on the first pooled pseudoreplicates </li><li>ppr2: Raw peaks called on the second pooled pseudoreplicates </li><li>repi: Raw peaks called on true replicate i </li><li>repi-pr1 : Raw peaks called on the first pseudoreplicate from replicate i </li><li>repi-pr2 : Raw peaks called on the second pseudoreplicates from replicate i </li><li>overlap : Overlapped peaks (filtered if blacklist exists) </li></ul></p><br></div><div style='float:left'><div id='table_Enrichment-QC-(Fraction-of-reads-in-peaks)'><b>Enrichment QC (Fraction of reads in peaks)</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>Fraction of Reads in Peak</th><td>0.167314</td></tr>
</table></div><div id='help-idr-FRiP'><p><ul><li>ppr: IDR peaks comparing pooled pseudo replicates </li><li>rep1-pr: IDR peaks comparing pseudoreplicates from replicate 1 </li><li>rep2-pr: IDR peaks comparing pseudoreplicates from replicate 2 </li><li>repi-repj: IDR peaks comparing true replicates (rep i vs. rep j) </li></ul></p><br></div><div style='float:left'><div id='table_Reproducibility-QC-and-Peak-Detection-Statistics-(Irreproducible-Discovery-Rate)'><b>Reproducibility QC and Peak Detection Statistics (Irreproducible Discovery Rate)</b>
<table border='1' style='margin-right:15px;margin-bottom:15px'>
<tr><th nowrap align=left>Nt</th><td>87838</td></tr>
<tr><th nowrap align=left>N1</th><td>&nbsp</td></tr>
<tr><th nowrap align=left>N2</th><td>&nbsp</td></tr>
<tr><th nowrap align=left>Np</th><td>0</td></tr>
<tr><th nowrap align=left>N optimal</th><td>87838</td></tr>
<tr><th nowrap align=left>N conservative</th><td>87838</td></tr>
<tr><th nowrap align=left>Optimal Set</th><td>rep1-rep2</td></tr>
<tr><th nowrap align=left>Conservative Set</th><td>rep1-rep2</td></tr>
<tr><th nowrap align=left>Rescue Ratio</th><td>0.0</td></tr>
<tr><th nowrap align=left>Self Consistency Ratio</th><td>1.0</td></tr>
<tr><th nowrap align=left>Reproducibility Test</th><td>pass</td></tr>
</table></div><div id='help-idr'><p><ul><li>N1: Replicate 1 self-consistent IDR 0.1 peaks (comparing two pseudoreplicates generated by subsampling Rep1 reads) </li><li>N2: Replicate 2 self-consistent IDR 0.1 peaks (comparing two pseudoreplicates generated by subsampling Rep2 reads) </li><li>Nt: True Replicate consistent IDR 0.1 peaks (comparing true replicates Rep1 vs Rep2 ) </li><li>Np: Pooled-pseudoreplicate consistent IDR 0.1 peaks (comparing two pseudoreplicates generated by subsampling pooled reads from Rep1 and Rep2 ) </li><li>Self-consistency Ratio: max(N1,N2) / min (N1,N2) </li><li>Rescue Ratio: max(Np,Nt) / min (Np,Nt) </li><li>Reproducibility Test: If Self-consistency Ratio >2 AND Rescue Ratio > 2, then 'Fail' else 'Pass' </li></ul></p><br></div><figure style='display: inline-block;'><img src='./peak/macs2/idr/true_reps/rep1-rep2/output_treatedMB6Pen_rep1-rep2.unthresholded-peaks.txt.png' width='800'><figcaption style='text-align: center;'><b>true reps (rep1-rep2)</b></figcaption></figure>&nbsp<div style='float:left'></div></div><br><br></body></html>